### ISSUE TITLE
Vault timeout not respected

### ISSUE THREAD

**Issue by kdpenner** [bitwarden/desktop#1127]
### Steps To Reproduce

Vault timeout is instantaneous regardless of settings. The following issues hold for the macOS desktop version:

https://github.com/bitwarden/mobile/issues/1557
https://github.com/bitwarden/mobile/issues/1363
https://github.com/bitwarden/browser/issues/1893

### Expected Result

Password should not be required for the timeout length selected in the settings.

### Actual Result

Password is prompted regardless of settings.

### Operating System

macOS

### Operating System Version

10.15.7

### Installation method

Mac App Store

### Build Version

1.29.0

**Comment by Hinton** [comment]
Hi,

What vault timeout option is specified in the settings?

**Comment by Wh1t3st4r** [comment]
"Indefinitely", after the most recent update that option did no change, it
would in contrast function as an "on restart" option, since whenever the
application is restarted, the password phase is activated to have access to
the vault.

**Comment by Hinton** [comment]
@Wh1t3st4r apologies, I was a bit confused and thought this was the browser extension. Desktop does not have an option for "Indefinitely" or "Immediately". Are you refering to Never? If so we're working on that in https://github.com/bitwarden/jslib/pull/682

**Comment by kdpenner** [comment]
I chose 4 hours.

1. Open desktop app; prompted for password
2. Log in
3. Close desktop app
4. Open desktop app again; prompted for password again

My expectation is that I will be prompted for a password only if 4 hours have passed between 1 and 4.

**Comment by Hinton** [comment]
@kdpenner That's the expected behavior, we don't want to store the password to disk.

**Comment by kdpenner** [comment]
@Hinton OK, I suggest rewording "Vault timeout" to something like "Vault timeout while open" or something like this.
